tests/views_integration/Tribe/Events/Views/V2/Partials/Month_View/Mobile_Events/Mobile_Day/Mobile_Event/CostTest.php

```diff
@@ -3,10 +3,10 @@
 
 use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
 
-class CtaTest extends HtmlPartialTestCase
+class CostTest extends HtmlPartialTestCase
 {
 
-	protected $partial_path = 'month/mobile-events/mobile-day/mobile-event/cta';
+	protected $partial_path = 'month/mobile-events/mobile-day/mobile-event/cost';
```

Comment:
use `$this->get_mock_event()` here, also fix tests to match the test names and cases of the other tests (list and day view)

Reply:
still missing test without cost